content/en/network_monitoring/netflow/_index.md

```diff
@@ -79,6 +79,23 @@ From the **Configuration** tab in NetFlow, click **Add Enrichment** to upload th
 
 {{< img src="network_device_monitoring/netflow/new_enrichment.png" alt="The New Enrichment Mapping modal in the Netflow configuration tab" width="80%" >}}
 
+### Reverse DNS private IP enrichment
+
+Enable Reverse DNS private IP enrichment to perform DNS lookups and find the domain name associated with any NetFlow source or destination IP address. When enabled, the Agent conducts reverse DNS lookups on source and/or destination IPs within private address ranges, enriching NetFlow records with the corresponding hostnames.
```

Comment:
"domain name" is mentioned initially, then "hostnames" at the end.  We should probably stick with one, IMO hostname.

"any NetFlow source or destination IP address" - saying "any" may be misleading since we only attempt reverse DNS lookups for private source/dest IP addresses (that are in the reserved private IP address ranges https://www.arin.net/reference/research/statistics/address_filters/)